docs/config.toml

```diff
@@ -28,6 +28,11 @@ menushortcutsnewtab = true # set true to open shortcuts links to a new tab/windo
 enableGitInfo = true
 operatorVersion = "5.0.3"
 centosBase = "centos8"
+imageCrunchyPostgresCentos = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.4-1"
+imageCrunchyPGBackrestCentos = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:centos8-2.35-0"
+imageCrunchyPGBouncerCentos = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos8-1.15-3"
+imageCrunchyExporterCentos = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.3-0"
```

Comment:
The exporter image would be ubi not centos. :wrench: ... maybe? Assuming we keep the OS in the name

Reply:
oh that was a copy paste error i'll fix that